chcemvediet/apps/accounts/admin.py

```diff
@@ -79,3 +89,20 @@ def get_queryset(self, request):
         queryset = queryset.select_related(u'user')
         queryset = queryset.select_undecided_emails_count()
         return queryset
+
+    def login_as_view(self, request, id):
+        admin_login_as = request.session.get(u'admin_login_as')
+        user = get_object_or_404(User, pk=id)
+        if not hasattr(user, u'backend'):
+            user.backend = u'chcemvediet.apps.accounts.backends.AdminLoginAsBackend'
```

Comment:
Toto hackovanie `user.backend` som nepochopil. Ved backend je nastaveny globalne v settingoch, naco ho tu overridujes?

Reply:
Ak som nenastavil explicitne `user.backend` tak  nefungoval login, ale uz to je nerelevantne.